### DISCUSSION TITLE
checkout page

### DISCUSSION THREAD

**Discussion by sadia-33** [woocommerce/woocommerce#51397] [Checkout Flow]
You can add some pre-made templates and designs in for easy customization

and more flexibility in designs such as custom colours, font, custom addition of placeholders

**Comment by alexflorisca** [comment]
Hi @sadia-33, thanks for the feedback. It would be useful to clarify a few things. I assume you mean the checkout block not shortcode? And could you be a bit more specific about the things you want to customize? We already provide a checkout template that you can use as a starting point.
